common/common.sh

```diff
@@ -2,6 +2,17 @@
 
 set -o errexit
 
+#install net-tools
+
+source /etc/os-release
+ID=${ID:-centos}
+LINUX_DISTRIB=$ID
+if [[ ${LINUX_DISTRIB} == 'centos' ]]; then
```

Comment:
see startup.sh#L23 
there are sections for installation of requried packages , just add the package there.